content/en/docs/tasks/administer-cluster/kubelet-config-file.md

```diff
@@ -30,7 +30,7 @@ it simplifies node deployment and configuration management.
 
 The subset of the Kubelet's configuration that can be configured via a file
 is defined by the `KubeletConfiguration` struct
-[here (v1beta1)](https://github.com/kubernetes/kubernetes/blob/release-1.10/pkg/kubelet/apis/kubeletconfig/v1beta1/types.go).
+[here (v1beta1)](https://github.com/kubernetes/kubernetes/blob/release-1.11/pkg/kubelet/apis/kubeletconfig/v1beta1/types.go).
```

Comment:
I guess you want to use param [here](https://github.com/kubernetes/website/blob/master/config.toml#L75).
```
[[params.versions]]
fullversion = "v1.11.0"
version = "v1.11"
githubbranch = "v1.11.0"
docsbranch = "release-1.11"
url = "https://kubernetes.io"
```

Then you don't need to hard-code the release version and modify it each release.

@chenopis the params are widely used in docs for such case, right?

Reply:
Done.
Let me know if it looks right, the only other use I saw of `docsbranch` was in `content/en/docs/concepts/configuration/assign-pod-node.md`, so I just copied that.

Reply:
Uh... I pushed why isn't this updating?

Reply:
@mtaufen I think this is the right way. Let @chenopis help confirm this :)